tests/test_radius_server.rb

```diff
@@ -90,9 +91,60 @@ def test_radiusserver_create_destroy_single
     refute_includes(Cisco::RadiusServer.radiusservers, id)
   end
 
+  def test_create_destroy_single_ipv6
+    id = '2004::3'
+    refute_includes(Cisco::RadiusServer.radiusservers, id)
+
+    server = Cisco::RadiusServer.new(id, true)
+    assert_includes(Cisco::RadiusServer.radiusservers, id)
+    assert_equal(Cisco::RadiusServer.radiusservers[id], server)
```

Comment:
You've got the params reversed on these `assert_equal` calls - should be `assert_equal(expected, actual)`